packages/eslint-plugin-next-on-pages/src/rules/no-unsupported-configs.ts

```diff
@@ -94,7 +94,7 @@ const rule: Rule.RuleModule = {
 	create: context => {
 		const code = context.getSourceCode();
```

Comment:
ASIDE: according to this doc the `getSourceCode()` function is deprecated: https://eslint.org/docs/latest/extend/custom-rules#:~:text=references%20to%20variables.-,getSourceCode(),-%3A%20(Deprecated%3A

Reply:
Yes thanks a bunch for the reminder, I noticed this too the other day, but doing it seemed like `sourceCode` wasn't a field of `context`... I might need to bump the eslint types package 🤔